include/SW_Flow_lib.h

```diff
@@ -64,18 +59,18 @@ void infiltrate_water_high(double swc[], double drain[], double *drainout, doubl
 
 void transp_weighted_avg(double *swp_avg, unsigned int n_tr_rgns, LyrIndex n_layers,
 						 unsigned int tr_regions[], double tr_coeff[], double swc[],
-						 SW_LAYER_INFO** lyr);
+						 SW_LAYER_INFO** lyr, LOG_INFO* LogInfo);
 
 void EsT_partitioning(double *fbse, double *fbst, double blivelai, double lai_param);
 
 void pot_soil_evap(unsigned int nelyrs, double ecoeff[], double totagb,
   double fbse, double petday, double shift, double shape, double inflec, double range,
   double width[], double swc[], double Es_param_limit, SW_LAYER_INFO** lyr,
-  double *bserate);
+  LOG_INFO* LogInfo, double *bserate);

```

Comment:
Is there a reason why the argument `LOG_INFO* LogInfo` does show up in virtually any position of the argument list across SOILWAT2 functions?

I feel it could help if it is treated differently than other arguments since it doesn't do anything with how the simulation works and is only about the user interface relaying messages, warnings, and errors. For instance, having it consistently as the last argument would seem to make the ordering of it much simpler (except for `LogError()` for which a first position does make most sense).